src/server/snapshot.cc

```diff
@@ -22,10 +22,6 @@
 #include "util/fibers/synchronization.h"
 
 using facade::operator""_MB;
-ABSL_FLAG(size_t, serialization_max_chunk_size, 0,
```

Comment:
that's not what I mean @BorysTheDev . I meant to keep the flag but to cache it into the global variable so that we won't access a flag with mutex/spinlock on every DbSlice operation.